### PR TITLE
Remove "#doctest: +SKIP"

### PR DESCRIPTION
**Pull request by gxyd**
Considering the PR #11652 (it has a large number of unnecessary commits). It wasn't easy to separate out commits, so I have created a clean PR to remove some `doctest: +SKIP`.

I'll close the other PR.

### CODE REVIEW

File: sympy/interactive/printing.py [hunk 1]
```diff
@@ -330,28 +330,28 @@ def init_printing(pretty_print=True, order=None, use_unicode=None,
...
     x**2 + x + y**2 + y
-    >>> init_printing(order='grevlex') # doctest: +SKIP
-    >>> str(y * x**2 + x * y**2) # doctest: +SKIP
+    >>> init_printing(order='grevlex')
+    >>> str(y * x**2 + x * y**2)
     x**2*y + x*y**2
-    >>> init_printing(order='old') # doctest: +SKIP
-    >>> str(x**2 + y**2 + x + y) # doctest: +SKIP
+    >>> init_printing(order='old')
+    >>> str(x**2 + y**2 + x + y)
```
Comment (ylemkimon) [inline sympy/interactive/printing.py]: Changing printing order affects subsequents doctests.

File: sympy/interactive/printing.py [hunk 2]
```diff
@@ -330,28 +330,28 @@ def init_printing(pretty_print=True, order=None, use_unicode=None,
...
-    >>> theta = Symbol('theta') # doctest: +SKIP
-    >>> init_printing(use_unicode=True) # doctest: +SKIP
-    >>> theta # doctest: +SKIP
-    \u03b8
-    >>> init_printing(use_unicode=False) # doctest: +SKIP
-    >>> theta # doctest: +SKIP
+    >>> theta = Symbol('theta')
+    >>> init_printing(use_unicode=True)
+    >>> theta
+    θ
```
Comment (ylemkimon) [inline sympy/interactive/printing.py]: AFAIK, Unicode doctests are not supported.

File: sympy/interactive/printing.py (hunk 1 above)
Comment (gxyd) [reply sympy/interactive/printing.py]: But they are already skipped, would it still lead to failure?

Comment (ylemkimon) [reply sympy/interactive/printing.py]: Doesn't this PR unskip it?

Comment (gxyd) [reply sympy/interactive/printing.py]: This PR unskips this doctest. But the ones after this are skipped. You said for 'subsequent' doctests, which wouldn't be a problem I guess.

Comment (ylemkimon) [reply sympy/interactive/printing.py]: I meant doctests in the subsequent files. `DocTestRunner` runs them in the same process.

Comment (gxyd) [reply sympy/interactive/printing.py]: Okay, I can now relate to it. I can see that now in `sympy/utilities/runtests.py`.

**Comment by gxyd** [comment]
I am closing this. Perhaps that is an issue to fix issue.

**Pull request closed without merging**